InterGraph/interaction_graph.py

```diff
@@ -0,0 +1,404 @@
+import os.path
+import pandas as pd
+import os
+import pickle
+import urllib
+import pandas as pd
+import numpy as np
+import networkx as nx
+import openbabel
+
+from tqdm import tqdm
+from rdkit import Chem
+from scipy.spatial.distance import cdist
+from itertools import product
+from openbabel import openbabel
+
+pdb_raw_d = "../data/" + "pdb_raw"
+
+
+def file_path():
+    global pdb_raw_d
+    if not os.path.exists("../data/" + "data"):
+        os.makedirs("../data/" + "data")
+    if not os.path.exists("../data/" + "pdb_raw"):
+        os.makedirs("../data/" + "pdb_raw")
+
+
+def get_pdb_components(pdb_id):
```

Comment:
can you add type hints here, in this case it would be:
` def get_pdb_components(pdb_id:str)->None:`

Reply:
and, maybe a short line that explains what this function does